README.md

```diff
@@ -13,14 +13,17 @@ A curated list of awesome things and projects built by Algerian developpers.
 ## Open Data
 - [Algerian Administrative Division](https://github.com/mohsenuss91/AlgerianAdministrativeDivision) - Wilayas - Daïras - baladiyahs
 - [Algerial Logos](https://github.com/mohsenuss91/AlgerianLogos) - Some Algerian official vectorized (.SVG) Logos
+- [algeria-cities](https://github.com/othmanus/algeria-cities) - A complete list of Algeria provinces and municipalities in csv, xlsx, sql, php and json formats.
```

Comment:
There is a default licence for projects that does not include one on github.

https://help.github.com/articles/open-source-licensing/#what-happens-if-i-dont-choose-a-license